docs/build/js-client/3-setting-up.md

```diff
@@ -1,94 +1,27 @@
 # Setting up project
 
-## Install dependencies
+## CLI install
```

Comment:
unfortunate title. reads to me that we are installing the CLI.

also there needs to be an intro at the very least explaining what we are setting up for.